gef.py

```diff
@@ -7216,22 +7220,27 @@ def __init__(self) -> None:
         super().__init__(complete=gdb.COMPLETE_LOCATION)
         return
 
+    @parse_arguments({"arena_address": ""}, {})
     @only_if_gdb_running
-    def do_invoke(self, argv: List[str]) -> None:
+    def do_invoke(self, *_: Any, **kwargs: Any) -> None:
         def fastbin_index(sz: int) -> int:
             return (sz >> 4) - 2 if SIZE_SZ == 8 else (sz >> 3) - 2
 
+        args = kwargs["arguments"]
+        if not gef.heap.main_arena:
+            err("Heap not initialized")
+            return
+
         SIZE_SZ = gef.arch.ptrsize
         MAX_FAST_SIZE = 80 * SIZE_SZ // 4
         NFASTBINS = fastbin_index(MAX_FAST_SIZE) - 1
 
-        arena = GlibcArena(f"*{argv[0]}") if len(argv) == 1 else gef.heap.main_arena
-
+        arena = GlibcArena(f"*{args.arena_address}") if args.arena_address else gef.heap.selected_arena
         if arena is None:
             err("Invalid Glibc arena")
             return
 
-        gef_print(titlify(f"Fastbins for arena {int(arena):#x}"))
+        gef_print(titlify(f"Fastbins for arena at {arena.addr}"))
```

Comment:
like here we'd just 

```suggestion
        gef_print(titlify(f"Fastbins for arena at {arena.addr:#x}"))
```

which has the advantage that we can for example use `format_address` if we want pretty (aligned) formatting

Reply:
resolved in 8454358